gui/templates/system/config_upload.html

```diff
@@ -3,7 +3,7 @@
 <tr>
   <td colspan="2">
    <p>{% trans "The system will reboot to perform this operation!" %}<p/>
-   <p style="color: red;">{% trans "All password will be reset in case your configuration bundle does not include the password secret seed." %}</p>
+   <p style="color: red;">{% trans "All passwords will be reset if your configuration bundle does not include the password secret seed." %}</p>
```

Comment:
I'd suggest s/your/the/

Is the rest of it clear enough?  Does the reader know what a "configuration bundle" is?

Reply:
My fix was just the grammar whilst keeping the rest of the sentence the same. 

As for your other comments, the terminology ("configuration bundle") is slightly different than what the rest of the UI and docs say (http://doc.freenas.org/11/system.html#general) -- eg "Save Config", "Upload Config" in the UI but "configuration", "configuration database" and "configuration backup" are used equally in the docs.  

The issue runs deeper than just this one-line fix, but I'll adjust this for now.